pub/htsget-openapi.yaml

```
@@ -20,55 +20,16 @@ paths:
       description: |
         Searches a pre-indexed object id.
       parameters:
-        - in: "path"
-          name: id
-          description: identifier of the read object
-          required: true
-          schema:
-            type: string
-        - in: "query"
-          name: format
-          description: File format, BAM (default), CRAM.
-          example: "BAM"
-          required: false
-          schema:
-            type: string
-        - in: "query"
-          name: class
-          description: Request different classes of data (such as header or body).
-          example: "header"
-          required: false
-          schema:
-            type: string
-        - in: "query"
-          name: referenceName
-          description: Reference sequence name
-          example: "chr1"
-          required: false
-          schema:
-            type: string
-        - in: "query"
-          name: start
-          description: The start position of the range on the reference, 0-based, inclusive.
-          example: "12312"
-          required: false
-          schema:
-            type: integer
-            format: int64
-            minimum: 0
-        - in: "query"
-          name: end
-          description: The end position of the range on the reference, 0-based exclusive.
-          example: "99999"
-          required: false
-          schema:
-            type: integer
-            format: int64
-            minimum: 0
+        - '$ref': '#/components/parameters/idParam'
```

It would be good to indicate in the ref name that this is a different kind of query from the others (it's the only `in: path` one), e.g.,

```suggestion
        - '$ref': '#/components/parameters/idPathParam'
```